### PR TITLE
Updates and connectivity status initial commit

### PR DESCRIPTION
**Pull request by roopavidhya**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/edge/Azure.ResourceManager.EdgeConnectivityStatuses/src/autorest.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Generated code configuration
+
+Run `dotnet build /t:GenerateCode` to generate code.
+
+``` yaml
+azure-arm: true
+csharp: true
+library-name: Edge
+namespace: Azure.ResourceManager.Edge
+require: https://raw.githubusercontent.com/roopavidhya/azure-rest-api-specs-pr/45dce3521d347333f5f41e21d5763f192abc4c79/specification/edge/resource-manager/Microsoft.Edge/connectivityStatuses/readme.md
```
Comment (live1206) [inline sdk/edge/Azure.ResourceManager.EdgeConnectivityStatuses/src/autorest.md:10]: @roopavidhya As mentioned in previous PR, we will consider reviewing the PR after the swagger is merged into public main.
And also, have you got the namespace approved for your service?

**Review by live1206 — CHANGES_REQUESTED** [review]

File: sdk/edge/Azure.ResourceManager.EdgeConnectivityStatuses/src/autorest.md (hunk 1 above)
Comment (roopavidhya) [reply sdk/edge/Azure.ResourceManager.EdgeConnectivityStatuses/src/autorest.md:10]: We have merged it to RPSaaSMaster. Is it also necessary to merge it into azure-rest-api-specs-pr/main?

Comment (roopavidhya) [reply sdk/edge/Azure.ResourceManager.EdgeConnectivityStatuses/src/autorest.md:10]: Regarding the namespace, yes. The swagger is ARMSignedOff

Comment (live1206) [reply sdk/edge/Azure.ResourceManager.EdgeConnectivityStatuses/src/autorest.md:10]: > We have merged it to RPSaaSMaster. Is it also necessary to merge it into azure-rest-api-specs-pr/main?

The swagger needs to be merged into azure-rest-api-specs/main, not private spec repo.

Comment (live1206) [reply sdk/edge/Azure.ResourceManager.EdgeConnectivityStatuses/src/autorest.md:10]: > Regarding the namespace, yes. The swagger is ARMSignedOff

Namespace approval is something like [this](https://github.com/Azure/azure-sdk-pr/issues/1220)

Comment (roopavidhya) [reply sdk/edge/Azure.ResourceManager.EdgeConnectivityStatuses/src/autorest.md:10]: Sorry this link isn't accessible for me 
> 
> Namespace approval is something like [this](https://github.com/Azure/azure-sdk-pr/issues/1220)

Comment (live1206) [reply sdk/edge/Azure.ResourceManager.EdgeConnectivityStatuses/src/autorest.md:10]: > Sorry this link isn't accessible for me
> 
> > Namespace approval is something like [this](https://github.com/Azure/azure-sdk-pr/issues/1220)

You can raise the request in the public repo, like [this](https://github.com/Azure/azure-sdk/issues/6473)

**Pull request closed without merging**